content/wiki/formatting_scheme.md

```diff
@@ -0,0 +1,25 @@
++++
+title = "Formatting scheme for Linebender projects"
++++
+
+## Copyright
+
+Code files should start with a copypright notice following this format:
+
+```rust
+// Copyright 2024 the XXXX Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+```
+
+The year should be the file's creation year.
+"XXXX" should be the project's name.
+The license depends on the project and the file type, but is usually "either Apache 2.0 or MIT License, whichever is least restrictive".
+
+## Rust
+
+Most Linebender projects files are written in Rust.
+For those files, the default formatting produced by `rustfmt` is fine.
```

Comment:
```suggestion
For those files, the formatting produced by `rustfmt` is required.
```

1. It's usually default, but may not be if we introduce `rustfmt.toml` like we have for Druid.
2. The use of *fine* implies there's a different option. However it's actually a strict requirement that the formatting matches what `rustfmt` produces.